hack/test/e2e-aws-prepare.sh

```diff
@@ -9,6 +9,12 @@ REGION="us-east-1"
 function cloud_image_upload() {
   CLOUD_IMAGES_EXTRA_ARGS=("--name-prefix=${1}" "--target-clouds=aws" "--architectures=amd64" "--aws-regions=${REGION}")
 
+  case "${1}" in
+    talos-e2e-nvidia-oss)
```

Comment:
need to invent another hack to make controlplanes (regular AMIs) also use force BIOS mode

Reply:
scratch that, the test is skipped, so it doesn't matter if I update all machines or just one